src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -80,14 +80,28 @@
   @end
 
   def main():
+    import argparse
+
     @# FIXME: Convert argv from strings to the correct types.
-    sample_{@apiMethod.name}(*sys.argv[1:])
+    parser = argparse.ArgumentParser()
+    @join param : sample.initCode.argDefaultParams
+      parser.add_argument('--{@param.identifier}')
```

Comment:
We should set the default value from the view model, preferably here where we define the flag. (We could also do it when we get the flags, manually substituting the default if no flag was provided. But the flag library should take be the one doing it....)

Reply:
Done. Setting everything to empty string for now.